app/code/community/TIG/MyParcel2014/Model/Api/MyParcel.php

```diff
@@ -805,6 +807,31 @@ protected function _getConsignmentData(TIG_MyParcel2014_Model_Shipment $myParcel
             $data['physical_properties']['weight'] = (int)$totalWeight;
         }
 
+        if($data['options']['package_type'] == self::DIGITAL_STAMP){
+            $totalWeight = 0;
+            $items = $myParcelShipment->getOrder()->getAllItems();
+            foreach($items as $item) {
+                if($item->getProductType() == 'simple') {
+                    $parentId = $item->getParentItemId();
+                    $weight   = floatval($item->getWeight());
+                    $qty      = intval($item->getQtyOrdered());
+
+                    if ( ! empty($parentId)) {
+                        $parent = Mage::getModel('sales/order_item')->load($parentId);
+
+                        if (empty($weight)) {
+                            $weight = $parent->getWeight();
+                        }
+                    }
+                    $weight      *= $qty;
+                    $totalWeight += $weight * 1000;
+                }
+            }
+
+            $data['physical_properties']['weight'] = $totalWeight;
```

Comment:
`physical_properties']['weight` is hierboven al gezet, dit hoef je dus helemaal niet te doen.

Reply:
Done!